ci/testnet-manager.sh

```diff
@@ -444,6 +459,26 @@ deploy() {
       fi
     )
     ;;
+  testnet-tds)
+    (
+      set -x
+      # TODO: Should we spread the few nodes around multiple zones?
+      # shellcheck disable=SC2068
+      # shellcheck disable=SC2086
+      NO_LEDGER_VERIFY=1 \
+      NO_VALIDATOR_SANITY=1 \
+        ci/testnet-deploy.sh -p tds-solana-com -C gce ${GCE_ZONE_ARGS[0]} \
+          -t "$CHANNEL_OR_TAG" -n "$GCE_NODE_COUNT" -c 1 -P -u \
+          -a tds-solana-com --hashes-per-tick auto \
+          ${skipCreate:+-e} \
+          ${skipStart:+-s} \
+          ${maybeStop:+-S} \
+          ${maybeDelete:+-D} \
+          -k 1000 -K /tmp/stakes.yml \
```

Comment:
`-k 1000` may be too small, that's enough for ~1 minute of voting.   I'd add many more zeros, this may be the boot issue

Reply:
That value was an arbitrary guess.  Is it worth pulling that data up to an env var (along with the /tmp/stakes.yml filepath?) so we can set it from buildkite job?